packages/portals/frontend/sirius-components-portals/src/hooks/usePortal.types.ts

```diff
@@ -68,6 +68,7 @@ export interface GQLRepresentationMetadata {
   id: string;
   label: string;
   kind: string;
+  iconURLs: string[];
```

Comment:
Why add `iconURLs` in type but do not retrieve it from the graphql subscription ?

Reply:
Done